upgrade-instructions.md

```diff
@@ -16,17 +16,36 @@ Changes to apply between 3.4.x and 3.5.x will be reported here.
 
 Change list:
 
+ - [RabbitMQ minimal version](#rabbitmq-minimal-version)
  - [Enforce usernames to be lower cased](#enforce-usernames-to-be-lower-cased)
  - [Cassandra keyspace creation configuration](#cassandra-keyspace-creation-configuration)
  - [UsersFileRepository](#usersfilerepository)
  - [ElasticSearch performance enhancements](#elasticsearch-performance-enhancements)
  - [JAMES-2703 Post 3.4.0 release removals](#james-2703-post-340-release-removals)
 
+#### RabbitMQ minimal version
```

Comment:
Did we wrote an ADR for that?

Reply:
that what? minimal version or new workqueue design?

Reply:
the design decision that leads to requiring Rabbit 3.8 (the usage of single active consumer)